overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -12876,7 +12876,7 @@
                     },
                     "ShowParticles:1": 1,
                     "Duration:3": 0,
-                    "Id:1": 63,
+                    "Id:1": 93,
```

Comment:
add an alternative, dont replace. this change will just change who has a broken questbook

Reply:
Icon might become wrong but quest is now completable at least